### ISSUE TITLE
clicking the "dock to ..." button with the inspector open breaks the dev tools

### ISSUE THREAD

**Issue by dylanb** [dequelabs/axe-firefox-devtools#12]
The UI "half-draws" the new panel, then stops and from that moment on dev tools is hosed. See the image below for the result when trying to change the docking from right to bottom.

![image](https://cloud.githubusercontent.com/assets/79245/8804004/07a8759a-2f98-11e5-96f2-ca5868aa7d75.png)

**Comment by dsturley** [comment]
```
ReferenceError: assignment to undeclared variable compiledRelatedListTemplate axe-panel.js:27:31
ReferenceError: assignment to undeclared variable compiledRelatedNodeTemplate axe-panel.js:28:31
ReferenceError: assignment to undeclared variable compiledFailureTemplate axe-panel.js:29:27
ReferenceError: assignment to undeclared variable compiledReasonsTemplate axe-panel.js:30:27
A promise chain failed to handle a rejection. Did you forget to '.catch', or did you forget to 'return'?
See https://developer.mozilla.org/Mozilla/JavaScript_code_modules/Promise.jsm/Promise

Date: Tue Jul 21 2015 12:01:42 GMT-0400 (EDT)
Full Message: 
Full Stack: JS frame :: resource://gre/modules/commonjs/toolkit/loader.js -> resource:///modules/devtools/framework/toolbox.js :: Toolbox.prototype.switchHost/< :: line 1466
JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: Handler.prototype.process :: line 867
JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: this.PromiseWalker.walkerLoop :: line 746
JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: this.PromiseWalker.scheduleWalkerLoop/< :: line 688
native frame :: <unknown filename> :: <TOP_LEVEL> :: line 0 toolbox.js:1466:0
```

**Comment by dsturley** [comment]
Should note this doesn't happen in Firefox Developer Edition.

**Comment by dsturley** [comment]
Firefox bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1075490

**Comment by dsturley** [comment]
I've managed to fix this by taking "inspiration" from [ember-inspector](https://github.com/emberjs/ember-inspector).

**Comment by dsturley** [comment]
Fixed with 0.3.0
